### ISSUE TITLE
Fix crash https://goto.bible/Acts.8_1/40072c4a5aba4022-01/de4e12af7f28f599-01

### ISSUE THREAD

**Issue by pmachapman** [pmachapman/GoTo.Bible#6]
There is an error in the server logs.

**Comment by pmachapman** [comment]
The error was:
```
System.ArgumentOutOfRangeException: Length cannot be less than zero. (Parameter 'length')
  at System.String.Substring(Int32 startIndex, Int32 length)
  at GoToBible.Engine.Renderer.RenderAsync(RenderingParameters parameters, Boolean renderCompleteHtmlPage)
  at GoToBible.Web.Server.Controllers.RenderPassageController.Post(RenderingParameters parameters, Boolean renderCompleteHtmlPage) in D:\source\repos\GoTo.Bible\GoToBible.Web\Server\Controllers\RenderPassageController.cs:line 89
```

**Comment by pmachapman** [comment]
Fixed in 6da402727f2ec9fde0f797cb8625b55dd2b6ae03
